biz.aQute.bnd/test/aQute/bnd/junit/JUnitFrameworkTest.java

```diff
@@ -0,0 +1,150 @@
+package aQute.bnd.junit;
+
+import java.net.URL;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.BundleEvent;
+import org.osgi.framework.BundleException;
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.osgi.service.packageadmin.PackageAdmin;
+import org.osgi.service.startlevel.StartLevel;
+import org.osgi.util.tracker.BundleTracker;
+
+import aQute.bnd.junit.JUnitFramework.BundleBuilder;
+import aQute.bnd.osgi.EmbeddedResource;
+
+@SuppressWarnings("deprecation")
+public class JUnitFrameworkTest extends Assert {
+	static {
+		System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "warn");
+	}
+	static JUnitFramework framework = new JUnitFramework();
```

Comment:
Why is this static and not per test method to avoid side effects between test methods?